### DISCUSSION TITLE
How to include a file in another directory?

### DISCUSSION THREAD

**Discussion by venpopov** [quarto-dev/quarto-cli#8487] [Q&A] [answered]
### Description

I want to include an Rmd file in many of my other files. This file is located in a folder lib at the base of the Quatro project. I currently do it like this:

````qmd
---
title: Parameter recovery simulation of individual differences
format: html
author: Ven Popov
date: Jan 29, 2024
---

{{< include ../../lib/_r-matrix-to-latex.Rmd >}}
````

is there a way to specify somewhere that includes should look in the lib folder, rather than using ../../ etc? I tried using `resource-path:`, but the synax isn't documented anywhere an I can't make it work.

**Comment by mcanouil** [comment, marked as answer]
Within a Quarto project you can use a leading `/` to indicate the file is relative to the root of the Quarto project: `{{< include /lib/_r-matrix-to-latex.Rmd >}}` shoudl work.

**Reply by venpopov** [reply]
Perfect, it works, thank you!
